src/createGoogleMapsMock.js

```diff
@@ -346,6 +346,9 @@ const createGoogleMapsMock = (libraries = []) => {
       AutocompleteService: jest.fn(() => ({
         getPlacePredictions: jest.fn(),
       })),
+      PlacesServiceStatus: jest.fn(() => ({
```

Comment:
Isn't that a constant?
```javascript
PlacesServiceStatus: {
      ERROR: 'ERROR', //???
      OK: 'OK',
    },
```

Reply:
Good point! I just copied the entire object over :-)